lti_consumer/lti_xblock.py

```diff
@@ -1026,6 +1026,15 @@ def lti_launch_handler(self, request, suffix=''):  # pylint: disable=unused-argu
 
         lti_consumer.set_custom_parameters(self.prefixed_custom_parameters)
 
+        for processor in self.get_parameter_processors():
+            try:
+                default_params = getattr(processor, 'lti_xblock_default_params', {})
+                lti_consumer.set_extra_claims(processor(self) or default_params)
```

Comment:
@pkulkark @nedbat is right on this one. 

The previous behavior was that all the default parameters were included and then overwritten by the ones provided by the parameter processor. To keep it working the same way, the code should be like this:
```suggestion
                default_params = getattr(processor, 'lti_xblock_default_params', {})
                lti_consumer.set_extra_claims(default_params)
                lti_consumer.set_extra_claims(processor(self) or {})
```

Maybe `set_extra_claims` is misleading here: it's not a `setter` since it **updates** a dictionary using python's `.update` method instead of overwriting content. Can you add a note in the method's description to clarify the behavior?

Reply:
@giovannicimolin Updated the code, rebased to latest master and updated the version :+1: